docker/Dockerfile.chronograf.prod

```diff
@@ -38,17 +29,20 @@ ARG INJECT_HEADER
 # Injecting strings into the html never went wrong. we use this for google tag manager
 ARG INJECT_BODY
 
-ENV PATH="/node/bin:${PATH}"
-RUN mkdir /node && \
-    curl -sL https://git.io/n-install | N_PREFIX=/node bash -s -- -q
-
 COPY . /repo/ui
 WORKDIR /repo/ui
 
+RUN apk \
+    --update \
+    --no-cache \
```

Comment:
is there a reason we wouldn't want to cache @drdelambre ?

Reply:
smaller image size mainly, but that doesn't matter since the final image is built from `nginx:stable-alpine` (see line 57/51 below)